x/dex/types/unit.go

```diff
@@ -0,0 +1,16 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var UnitToMultiplier = map[Unit]sdk.Dec{
+	Unit_STANDARD: sdk.OneDec(),
+	Unit_MILLI:    sdk.NewDec(1_000),
+	Unit_MICRO:    sdk.NewDec(1_000_000),
+	Unit_NANO:     sdk.NewDec(1_000_000_000),
+}
+
+func ConvertDecToStandard(unit Unit, dec sdk.Dec) sdk.Dec {
```

Comment:
if this is only used by test package can we move this to unit_test

Reply:
i deleted it since it doesn't seem to be used anywhere